github.md

```diff
@@ -22,3 +32,73 @@ you should do:
+
+## Github and JIRA
+
+In general, if you are creating code on GitHub, commenting and review should happen there, with the code. If you are also using JIRA, you should add a comment with a link to the PR to the story in JIRA and move the story to the Review column to indicate that the PR is under review.
+
+(If changes are not commited to GitHub - like with Wordpress configurations - the comments and review should happen in JIRA; Mention reviewers by name in a comment on the ticket, what they should be reviewing, where they can view the changes, and then move the story to the Review column.)
+
+- - -
+
+## GitHub basics
+
+This process assumes you have: a terminal program (like iTerm with oh-my-zsh), an editor program (like Sublime Text or Atom), a github account and access to the MIT Libraries repos, and have already cloned the repos to your local machine. (For help with those steps, contact one of the MITLib devs.)
+
+*Start fresh*: Checkout master branch
+```git checkout master```
+
+*Get latest*: Get the latest changes that have been merged 
+```
+git fetch
+git merge
+```
+
+*Create a branch*: Create a new branch starting with master (or the branch you want to base your work on): This is the clean separate space where you will put your work.
+```git checkout -b [branchname]```
+
+*Do stuff*: open the repo in your editor and write code or make edits that relate to the fix/feature/experiment you are working on.
+
+*Commit your changes regularly*: once you have a few changes that add up to something (eg. 'added new js function to load rss' or 'added new css to support adjusted mini-cal alignment') or if you have to stop for a bit or switch to something else, add the files you are ready to commit, then commit them to your branch and push them to github.
+
+```
+git add [filename] 
+git add [filename] 
+...
+git commit
+[add a description of what's in the commit]
+git push
+```
+
+*Do more stuff*: keep working and committing (repeat steps 4 and 5) until your work for that fix/feature/experiment is as complete as you can make it.
+
+*Make a PR*: go to github and create a new PR for your branch against master/prod branch (see Pull Request Reviews above). 
+
+*Respond to PR feedback*: as feedback comes in, create additional commits to address and push them up.
+
+*Get thumbs*: once each reviewer is done with their review and happy with the code that will be merged, they should approve the PR and give a thumbs up  :+1:
+
+*Squash commits (optional)*: before merging, squash all your work into logical commits - or one single commit - that summarizes the changes.
+
+*Merge the PR*: merge the PR on github and delete the branch.
+
+*Pull latest master*: Return to your terminal, checkout the master branch and pull the latest changes.
+```
+git checkout master
```

Comment:
again, I do it differently. I suspect your way probably works :)

For reference, I do `git fetch` then `git checkout origin/master` to avoid needing a local `master`.